1-exercises/E-for-loop/exercise2.js

```diff
@@ -28,6 +28,21 @@ const AGES = [
 
 // TODO - Write for loop code here
 
+for(i = 0;i <= WRITERS.length;i++ ){
```

Comment:
The comparison should be `i < WRITERS.length`, because array start counting from 0. With the current code, you will get an undefined for the last line.